code_annotations/config_and_tools/config/feature_toggle_annotations.yaml

```diff
@@ -11,17 +11,15 @@ annotations:
   feature_toggle:
     - ".. toggle_name:":
     - ".. toggle_implementation:":
-        choices: [ExperimentWaffleFlag, WaffleFlag, WaffleSample, WaffleSwitch, CourseWaffleFlag, ConfigurationModel, DjangoSetting]
+        choices: [WaffleFlag, WaffleSwitch, CourseWaffleFlag, ExperimentWaffleFlag, ConfigurationModel, DjangoSetting]
```

Comment:
Could be a separate PR, but we may want to update to SettingToggle and SettingDictToggle.

Reply:
I'll add this now to avoid back-and-forth PRs.